src/var/www/hsmm-pi/Controller/SystemController.php

```diff
@@ -4,7 +4,7 @@ class SystemController extends AppController {
 
 	public function reboot() {
 
-		file_put_contents('/var/data/hsmm-pi/reboot', time());
+		exec('sudo /sbin/shutdown -r +1');
```

Comment:
A future refactor may include creating a new user named "hsmm-pi" (or something) and provide this user with sufficient privileges to allow them to run things without needing to use sudo everywhere.